<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassesWidget</class>
 <widget class="QWidget" name="ClassesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>836</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="qf::qmlwidgets::Splitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Classes</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="qf::qmlwidgets::TableViewToolBar" name="tblClassesTB" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkUseAllMaps">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>When checked, all the maps defined for class will be used during drawing</string>
           </property>
           <property name="text">
            <string>Use all maps</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="ClassesTableView" name="tblClasses"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Codes</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="lblCourseCodes">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="qf::qmlwidgets::TableViewToolBar" name="tblCourseCodesTB" native="true"/>
       </item>
       <item>
        <widget class="qf::qmlwidgets::TableView" name="tblCourseCodes"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qf::qmlwidgets::TableView</class>
   <extends>QTableView</extends>
   <header location="global">qf/qmlwidgets/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>qf::qmlwidgets::TableViewToolBar</class>
   <extends>QWidget</extends>
   <header location="global">qf/qmlwidgets/tableviewtoolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qf::qmlwidgets::Splitter</class>
   <extends>QSplitter</extends>
   <header location="global">qf/qmlwidgets/splitter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClassesTableView</class>
   <extends>QTableView</extends>
   <header location="global">plugins/Classes/src/classestableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
